src/tests/ftest/container/list_containers.py

```diff
@@ -43,8 +43,8 @@ def create_list(self, count, pool_uuid, expected_uuids):
                 self.daos_cmd.get_output("container_create", **kwargs)[0])
         expected_uuids.sort()
         # daos pool list-cont returns container UUIDs as a list.
-        data = self.daos_cmd.pool_list_cont(**kwargs)
-        actual_uuids = data["uuids"]
+        data = self.daos_cmd.container_list(**kwargs)
+        actual_uuids = [uuid_label["UUID"] for uuid_label in data["response"]]
```

Comment:
Until we start creating containers with labels should we be verifying the rest of the `daos container list` command, e.g. `data["Label"] == "container_label_not_set"`